spec/models/profile_spec.rb

```diff
@@ -299,9 +299,39 @@
 
       expect(profile).to be_active
     end
+
+    context 'when the initiating_sp is the IRS' do
+      it 'logs an attempt event' do
+        sp = create(:service_provider, :irs)
+        profile = create(
+          :profile, user: user, active: false, fraud_review_pending: true,
+                    initiating_service_provider: sp
```

Comment:
Not sure if this was my reinstall of RubyMine or `rubocop -a`, but this feels like a spectacularly random level of indentation. I guess it's trying to keep the `:profile` class distinct and then line up all the parameters, but :eye-twitch:

Reply:
I would manually unindent this
```suggestion
          initiating_service_provider: sp
```